.travis.yml

```diff
@@ -1,23 +1,8 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
+  - 2.1.10
+  - 2.2.5
+  - 2.3.1
```

Comment:
One ruby version is sufficient. We'll probably want to add different solidus versions and testing against all versions of both is excessive.